packages/block-library/src/social-link/test/index.native.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, initializeEditor, waitFor, within } from 'test/helpers';
+/**
+ * WordPress dependencies
+ */
+import { registerCoreBlocks } from '@wordpress/block-library';
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+
+const unregisterBlocks = () => {
+	const blocks = getBlockTypes();
+
+	blocks.forEach( ( { name } ) => unregisterBlockType( name ) );
+};
+
+describe( '<SocialLinkEdit/>', () => {
+	beforeAll( () => {
+		registerCoreBlocks();
+	} );
+
+	afterAll( () => {
+		unregisterBlocks();
+	} );
+
+	/**
+	 * GIVEN an EDITOR is displayed;
+	 * WHEN a SOCIAL ICONS BLOCK is selected from the BLOCK INSERTER;
+	 */
+	it( 'should display WORDPRESS, FACEBOOK, TWITTER, INSTAGRAM by default.', async () => {
+		// Arrange
+		const subject = await initializeEditor( {} );
```

Comment:
The staging in each of these tests is purposefully redundant in an effort to keep the code easy to understand in isolation. I don't like DRY code in test suites personally, but the thought crossed my mind about checking for existing utils or creating one. It just doesn't feel worth it at the moment.